alpha-build-machinery/make/targets/golang/verify-update.mk

```diff
@@ -26,6 +26,16 @@ verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
 
+verify-sec: GOSEC_GOPATH :=$(shell mktemp -d)
```

Comment:
@tnozicka  wondering, we also create `_output` now. Shouldn't we put those things also there? Temp files created from Makefiles have a code smell.

Reply:
So... should I remove the mktemp call?

Reply:
Not sure, this is pretty tricky and we currently do it in several ways. We checkpoint some tool versions in our build images but that doesn't have effect on local dev so we just assume devs get compatible version. (e.g. glide just errors out and informs the user to get a compatible version). Some tools like bindata we vendor and then create tmpdir to install them into but that's for those that need to be coupled to the project. Vendoring it means common deps issues.

Maybe the least controversial in this case is to output an error message e.g. with the `go get` command for user to install the tool, possibly check version as well. If this get hooked into CI we will pre-install in the builder image.